sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2859,7 +2859,13 @@ def _provision(
                 # TODO(suquark): once we have sky on PyPI, we should directly
                 # install sky from PyPI.
                 local_wheel_path, wheel_hash = wheel_utils.build_sky_wheel()
-            backoff = common_utils.Backoff(_RETRY_UNTIL_UP_INIT_GAP_SECONDS)
+                # The most frequent reason for the failure of a provision
+                # request is resource unavailability instead of rate
+                # limiting; to make users wait shorter, we do not make
+                # backoffs exponential.
+                backoff = common_utils.Backoff(
+                    initial_backoff=_RETRY_UNTIL_UP_INIT_GAP_SECONDS,
```

Comment:
Just tried a few clouds and it seems we can also reduce this `_RETRY_UNTIL_UP_INIT_GAP_SECONDS` to `30`. : )